coding-concepts.js

```diff
@@ -37,23 +37,23 @@ const languages = ["JavaScript", "Ruby", "Python", "C++"]
 const index = 1
 // console.log(languages[index])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: "Javascript"
+// b) Verify and explain: My answer was incorrect because I chose index [1] where it should have been index [0] since it starts from [0].
 
 
 // --------------------4) What will this log?
 
 const weekendDays = ["saturday", "sunday"]
 // console.log(weekendDays.toUpperCase())
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: "SATURDAY" "SUNDAY"
+// b) Verify and explain: My answer was incorrect because this would need to be a string to console.log the .toUpperCase() and output "SATURDAY, SUNDAY". Since it was an array, this output would not work and we would need to find a different way to use .toUpperCase.
 
 
 // --------------------5) What will this log?
 
 const dataTypes = ["number", "string", "Boolean"]
 // console.log(typeof dataTypes.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: 3
+// b) Verify and explain: My answer was incorrect because it was outputting what datatype such as string, number, boolean, object and function, not the length of the variable "dataTypes"
```

Comment:
Excellent explanations in this section!  Some of these problems are definitely tricky and the output is way different than we often expect.